game/static/game/css/board.css

```
@@ -1363,6 +1371,17 @@ body {
+
+  .m-fab {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    position: fixed;
+    top: 14px;
+    right: 12px;
+    z-index: 10000;
+  }
+  .m-fab-btn {
 @media (max-width: 768px) {
```

Stylelint (17.12.0)

[error] Unclosed block (CssSyntaxError)

(CssSyntaxError)